pytrainer/extensions/osm.py

```diff
@@ -3,20 +3,17 @@
 # TODO: Add Google satellite images layers ?
 # TODO: Create map class/interface that osm/googlemaps will inherit from ?
 
-import os
-import re
+import html
 import logging
-import requests         # for downloading cached versions of openlayers.js and openstreetmaps.js
-import time             # Used for checking if local cached file is current
-
-import cgi as html
+import os
+import time  # Used for checking if local cached file is current
 
-from pytrainer.lib.gpx import Gpx
 import pytrainer.lib.points as Points
+import requests  # for downloading cached versions of openlayers.js and openstreetmaps.js
```

Comment:
Was it intentional to move this import down here between the `import as` and `from`s?

Reply:
After this change it's still unclear why the `import requests` was moved. I think it should be in the same place as before, between `o` and `t` alphabetically.

I think the indentations for the comments fulfilled a purpose too.

Reply:
After this change it's still unclear why the `import requests` was moved. I think it should be in the same place as before, between `o` and `t` alphabetically.

I think the indentations for the comments fulfilled a purpose too.